doc/databinding/index.md

```diff
@@ -2,19 +2,21 @@
 ---
 # Data Binding
 
+A complete example Tabris.js application that uses data binding comprehensively can be found [here]([here](https://github.com/eclipsesource/tabris-js-reddit-viewer).
+
 ## Setup
 
-Tabris provides a [JSX](../JSX.md)/[decorators](http://www.typescriptlang.org/docs/handbook/decorators.html)-based declarative data binding via the `'tabris-decorators'` extension, which has to be installed in addition to the `tabris` module:
+Tabris provides [JSX](../JSX.md)/[decorators](http://www.typescriptlang.org/docs/handbook/decorators.html)-based declarative data binding via the `'tabris-decorators'` extension, which has to be installed in addition to the `tabris` module:
 
 ```
 npm install tabris-decorators
 ```
 
- [**This extension works only with TypeScript/JSX projects**](../typescript.md#setup). Mixed projects work as well, but the modules using the extension need to be `.ts` or `.tsx` files. It's strongly recommended to familiarize yourself with the [basic decorators syntax](https://www.typescriptlang.org/docs/handbook/decorators.html#class-decorators) if you aren't already.
+ [This extension is optimized for TypeScript/JSX projects](../typescript.md#setup), but JavaScript projects work as well as long is they use the TypeScript compiler to provide JSX and decorators support. It's strongly recommended to familiarize yourself with the [basic decorators syntax](https://www.typescriptlang.org/docs/handbook/decorators.html#class-decorators) if you haven't already.
```

Comment:
Maybe provide information what needs to be configured in the tsconfig.json or a minimal tsconfig.json?

Reply:
The link provides this.